.github/workflows/ubuntu_18_04.yml

```diff
@@ -0,0 +1,73 @@
+name: Kinetic & Melodic ROS CI
+
+on: [push]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os_distro: [[ubuntu-16.04, kinetic], [ubuntu-18.04, melodic]]
```

Comment:
```suggestion
        os_distro: [[ros:kinetic-ros-core, kinetic], [ros:melodic-ros-core, melodic]]
```

Reply:
The runs-on has to be one of these https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on

```
No runner matching the specified labels was found: ros:melodic-ros-core.
```
https://github.com/lucasw/ros1_lifecycle/actions/runs/828276374

I think there are options for self-hosted runners to get around this, or docker commands can be embedded in the actions (but possibly would be much slower than installing packages in many cases).

(also the answer to my comment elsewhere is here- `runs-on` ubuntu-16.04 is only supported until September 2021 but fine to keep running it until then)

Reply:
@iluetkeb seems like @lucasw cannot make the suggested change if we want this to work as-is. I believe merging this as-is would still be helpful. If you think otherwise, would you share some inputs? lucas suggested couple ideas in the above comment